recipe/0001-Link-to-abseil_dll.patch

```diff
@@ -1,17 +1,8 @@
-From b2828440f9930c76e42235ddccfa0954d58a84ac Mon Sep 17 00:00:00 2001
-From: "Uwe L. Korn" <uwe.korn@quantco.com>
-Date: Tue, 4 Jul 2023 11:29:21 +0200
-Subject: [PATCH] Link to abseil_dll
-
----
- CMakeLists.txt | 2 +-
```

Comment:
i think you should attempt to keep the attribution of this patch.

Reply:
@xhochy Is there some official guideline to include such a header in the patch?

Reply:
What is the reason for this @hmaarrfk? The original author can be determined through git ultimately.